apps/tenant-management-webapp-e2e/src/integration/calendar.feature

```diff
@@ -16,4 +16,34 @@ Feature: Calendar
     When the user selects the "Calendar" menu item
     Then no critical or serious accessibility issues on "calendar overview page"
     When the user selects "Calendars" tab for "Calendar"
-    Then no critical or serious accessibility issues on "calendar calendars page"
\ No newline at end of file
+    Then no critical or serious accessibility issues on "calendar calendars page"
+
+  @TEST_CS-330 @REQ_CS-580 @REQ_CS-1652 @REQ_CS-1709 @regression
+  Scenario: As a tenant admin, I can add, edit and delete calendars in the tenant admin view
+    Given a tenant admin user is on tenant admin page
+    When the user selects the "Calendar" menu item
+    Then the user views the "Calendar service" overview content "The calendar service provides"
+    When the user clicks Add calendar button on overview tab
+    Then the user views Add calendar modal
+    When the user clicks Cancel button in Add calendar modal
+    Then the user views Calendar tab table header
+    When the user clicks Add calendar button
```

Comment:
Please add "on calendars page" to be more specific.

Reply:
added